src/sdk/pynni/nni/nas/pytorch/callbacks.py

```diff
@@ -51,4 +54,23 @@ def __init__(self, checkpoint_dir, every="epoch"):
         os.makedirs(self.checkpoint_dir, exist_ok=True)
 
     def on_epoch_end(self, epoch):
-        self.trainer.export(os.path.join(self.checkpoint_dir, "epoch_{}.json".format(epoch)))
+        dest_path = os.path.join(self.checkpoint_dir, "epoch_{}.json".format(epoch))
+        _logger.info("Saving architecture to %s", dest_path)
+        self.trainer.export(dest_path)
+
+
+class ModelCheckpoint(Callback):
+    def __init__(self, checkpoint_dir, every="epoch"):
+        super().__init__()
+        assert every == "epoch"
```

Comment:
support only one specific value? so why we make it a parameter?

Reply:
The roadmap is to support checkpoint on every mini-batch. But it's not implemented yet.

Reply:
please remove, because it is confusing.

Reply:
fixed in another PR.